docs/detections/rules-ui-manage.asciidoc

```diff
@@ -82,6 +83,24 @@ image::images/install-prebuilt-settings.png[]
 [role="screenshot"]
 image::images/install-prebuilt-rules.png[]
 
+[float]
+[[rule-prerequisites]]
+=== Confirm rule prerequisites
+
+Many Elastic prebuilt rules are designed to work with specific Elastic integrations and data fields. These prerequisites are identified in the *Related integrations* and *Required fields* fields on a rule's details page (*Detect* -> *Rules*, then click a rule's name). *Related integrations* also displays each integration's installation status and includes links for installing and configuring the listed integrations. 
```

Comment:
The sidebar menu structure has been changed and this is not correct anymore: `*Detect* -> *Rules*`.

Reply:
```suggestion
Many Elastic prebuilt rules are designed to work with specific Elastic integrations and data fields. These prerequisites are identified in the *Related integrations* and *Required fields* fields on a rule's details page (*Manage* -> *Rules*, then click a rule's name). *Related integrations* also displays each integration's installation status and includes links for installing and configuring the listed integrations. 
```

Reply:
In 8.4 the page is hidden under this `Manage` menu:

<img width="619" alt="Screenshot 2022-08-23 at 21 41 27" src="https://user-images.githubusercontent.com/7359339/186251486-0d5590bc-0066-42de-8f22-c89691805ca1.png">